program-runtime/src/invoke_context.rs

```diff
@@ -103,6 +103,7 @@ pub struct ThisInvokeContext<'a> {
     sysvars: &'a [(Pubkey, Vec<u8>)],
     logger: Rc<RefCell<dyn Logger>>,
     compute_budget: ComputeBudget,
+    current_compute_budget: ComputeBudget,
```

Comment:
What is `current_compute_budget` and how is it different from `compute_budget` ?

Reply:
Since the bump is only for a single instruction `compute_budget` which is the original is used to reset `current_compute_budget` for additional instructions that don't meet the bump criteria.

Reply:
If no further concerns I'll commit it